colossalai/pipeline/schedule/interleaved_pp.py

```diff
@@ -184,7 +206,15 @@ def forward_step(
 
         # for the first stage, input_obj is None
         # for the non-first stage, input_obj is the output of the previous stage and it's must be a dict
-        output_obj = model_forward(model_chunk[model_chunk_id], micro_batch, input_obj)
+
+        if isinstance(model_chunk, ModuleList):
+            output_obj = model_forward(model_chunk[model_chunk_id], micro_batch, input_obj)
+        else:
```

Comment:
When will it be normal module?

Reply:
It is actually a normal module for the bert finetune example, because we pass in the entire model for each stage (not a list of modules that the current stage is responsible for), just that the layers which the current stage is not responsible for have their weights set to none. 

It is a module list for the previously written test_interleaved.py script.